scripts/tilesets/wargus/summer.lua

```diff
@@ -10,7 +10,7 @@
 --
 --      summer.ccl - Define the summer tileset.
 --
---     (c) Copyright 2000-2023 by Lutz Sammer, Jimmy Salmon and Alyokhin
+--     (c) Copyright 2000-2024 by Lutz Sammer, Jimmy Salmon and Alyokhin
```

Comment:
even 2025 ;-)

Reply:
To be honest, I've been doing these highdowns since 2023 or even 2022. 😬